GoogleForCreators/web-stories-wp
# Dashboard: X new Templates Badge

## Context
Part of template sorting and notifying users of new templates


## Summary
- Adds sorting to templates so newest created templates appear first
- Adds new badge count based on templates < 60 days old


## Relevant Technical Choices
Reformats how were setting creationDates in our templates so that we can see how many new templates we have without pulling in all of the templates. 

Also handles all dates with our `@web-stories-wp/date` package.


## To-do
NA


## User-facing changes
Users will now see a badge on the dashboard nav that indicate all templates that were added within the past 60 days:
<img width="330" alt="Screen Shot 2021-08-18 at 12 26 52 PM" src="https://user-images.githubusercontent.com/35983235/129944315-0e280261-ab94-4ace-a18c-3c229a04d128.png">

Also templates will be sorted by most recently created.


## Testing Instructions
Open the dashboard and see that theres a badge indicating how many new templates there are. Then navigate to the templates and see that they're sorted by newest to oldest.


### QA
- Open Dashboard
- Should see a badge indicating 43 new templates created on the main navigation link for `Explore Templates`
- Click on `explore templates` to navigate to the explore templates view
- See that the original 8 templates are the last templates in the explore templates view





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8465 
Fixes #8464
